packages/luna/src/components/Lessons/PastLessonsTable.tsx

```diff
@@ -48,6 +52,14 @@ export type Props = {
   onRebook?: (tutorId: number) => void;
   onSendMessage?: (lessonId: number, members: number[]) => void;
   retry?: Void;
+  /**
+   * Function used to fetch more data list
+   */
+  more?: Void;
+  /**
+   * More button loading indicator
+   */
+  loadingMore?: boolean;
 };
 
 export const PastLessonsTable: React.FC<Props> = ({
```

Comment:
The changes done to this component makes it a little bit complex. The naming also is misleading becuase on mobile there is not table. The believe the changes done in this component should be in a stand alone component. In nova, you can switch between them based the current screen size. You can call the component `PastLessonsSummary` 


```tsx
mq.lg ? <PastLessonsTable /> : <PastLessonsSummary>
```